evennia/accounts/accounts.py

```diff
@@ -195,6 +196,9 @@ class DefaultAccount(AccountDB, metaclass=TypeclassBase):
 
     objects = AccountManager()
 
+    # Used by account.create_character() to choose default typeclass for characters.
+    player_character_typeclass = settings.BASE_CHARACTER_TYPECLASS
```

Comment:
best rename this to `default_character_typeclass` to better reflect its purpose.